File: CPP/LH05/DecimalTOBinary.cpp
#include <iostream>
using namespace std; 
int main(){
    int num = 8; 
    int rem = 0; 
    
    while(num){
        int r = num%2; 
        rem = rem*10+r;
        num = num/2; 
    }
    cout<<rem<<endl;
}

File: Java/ApnaJava/FunctionMethod/DecimalToBinary.java
package FunctionMethod;
import java.util.*;

public class DecimalToBinary {
    public static int DecToBin(int num){
        int dig = 0;
        int pow = 0; 

        while(num > 0){
            int rem = num % 2;
            dig = dig + (rem * (int)Math.pow(10, pow));
            pow++;
            num /= 2;
        }
        return dig;
    }
    public static void main(String[] args){
        System.out.println(DecToBin(4));
        
    }
    
}
